### PR TITLE
logical definition photoreceptor update

### PR DESCRIPTION
**Pull request by JABelfiore**
logical definition photoreceptor update and update text description to match

### CODE REVIEW

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -5463,14 +5463,14 @@ SubClassOf(obo:CL_0000209 obo:CL_0002076)
...
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MESH:D010786") obo:IAO_0000115 obo:CL_0000210 "A cell specialized in detecting light stimuli that are involved in visual perception.")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000210 "BTO:0001060")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000210 "CALOHA:TS-0868")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000210 "FMA:86740")
 AnnotationAssertion(rdfs:label obo:CL_0000210 "photoreceptor cell")
 AnnotationAssertion(foaf:depiction obo:CL_0000210 "https://www.swissbiopics.org/api/image/Photoreceptor_cell.svg"^^xsd:anyURI)
-EquivalentClasses(obo:CL_0000210 ObjectIntersectionOf(obo:CL_0000540 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0050962)))
-SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0000210 obo:CL_0000006)
+EquivalentClasses(obo:CL_0000210 ObjectIntersectionOf(obo:CL_0000540 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0050908)))
+SubClassOf(obo:CL_0000210 obo:CL_0008028)
```
Comment (anitacaron) [inline src/ontology/cl-edit.owl:5473]: I think this change caused the inferred equivalence, which is not allowed and it's the reason the QC is failing. :) 

```bash
2024-08-01 12:28:49,500 ERROR org.obolibrary.robot.ReasonOperation - Only equivalent classes that have been asserted are allowed. Inferred equivalencies are forbidden.
2024-08-01 12:28:49,502 ERROR org.obolibrary.robot.ReasonOperation - Equivalence: <http://purl.obolibrary.org/obo/CL_0000488> == <http://purl.obolibrary.org/obo/CL_0000210>
```

**Review by anitacaron — DISMISSED** [review]

**Comment by dosumis** [comment]
Fixed in GO.  If we are using GO nightly release, will be fixed here if we update GO in this branch on Monday.

File: src/ontology/cl-edit.owl (hunk 1 above)
Comment (AvolaAmg) [reply src/ontology/cl-edit.owl:5473]: I will approve the PR after Anita's changes resolved :)

Comment (anitacaron) [reply src/ontology/cl-edit.owl:5473]: David said he fixed that in GO. We're waiting for a GO release and will update the imports then.

**Comment by dosumis** [comment]
@aleixpuigb could you work with @JABelfiore to update the GO import today?

**Comment by dosumis** [comment]
New GO release on 2024-09-08, so update to GO should now fix.

**Comment by dosumis** [comment]
@aleixpuigb  - could you update GO Import.  Should fix.  Thanks!

**Comment by aleixpuigb** [comment]
#gogoeditdiff

**Review by aleixpuigb — APPROVED** [review]
It looks good to me.

**Comment by dosumis** [comment]
Good that it's finally passing, but...

@aleixpuigb  - can you look into conflicts? I think definitions.owl can be resolved by choosing version on main.  Maybe merged_import needs regen again?  We should aim to merge down ASAP as soon as fixed.  I can over-ride review requirements if needed.

**Review by aleixpuigb — APPROVED** [review]

**Pull request merged**